rogue-thi-app/components/page/AppTabbar.js

```diff
@@ -28,29 +29,35 @@ export default function AppTabbar () {
 
       <Navbar fixed="bottom" className={[styles.navbar, 'mobile-only']}>
         <Nav className={['justify-content-around', styles.nav]}>
+          <Nav.Item>
+            <Nav.Link onClick={() => router.replace('/library')} className={[styles.tab, router.pathname === '/library' && styles.tabActive]}>
+              <FontAwesomeIcon icon={faBook} className={styles.icon} />
+              Bibliothek
```

Comment:
Can you move the tabbar changes to a future PR? I think that design element isn't finished yet

Reply:
Done.